plugins/eval.py

```diff
@@ -56,10 +59,47 @@ def code_print(*args, sep=" ", end="\n", file=fp, flush=False):
         mk_code_print(fp)(repr(exc))
         del tb
 
+    def chunk(l, n): 
+        chunk_pos = range(0, len(l), n)
+        return [l[i:i+n] for i in chunk_pos]
+
+    # greedily concatenate strings into groups of at most a certain length
+    def chunk_concat(l, n): 
```

Comment:
Maybe rewrite the function a little so that `l` can be a generator?

Reply:
See 3c66142.